revolv/project/models.py

```diff
@@ -384,6 +397,15 @@ def partial_completeness(self):
         ratio = self.amount_donated / float(self.funding_goal)
         return min(ratio, 1.0)
 
+    @property
+    def percent_complete(self):
+        """
+        :return: an int between 0 and 100, representing the completeness of this
```

Comment:
We should note in this docstring that the percent will be a floor of the actual percent (e.g. 97.5 => 97)